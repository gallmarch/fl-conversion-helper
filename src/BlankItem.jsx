import React from 'react';

export default function BlankItem(props) {
  return <li id={props.id} className="empty-icon flch-empty-icon" />;
}
