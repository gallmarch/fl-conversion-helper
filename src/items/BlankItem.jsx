import React from 'react';
import PropTypes from 'prop-types';

export default function BlankItem(props) {
  return <li id={props.id} className="empty-icon flch-empty-icon" />;
}

BlankItem.propTypes = {
  id: PropTypes.string,
};

BlankItem.defaultProps = {
  id: undefined,
};
