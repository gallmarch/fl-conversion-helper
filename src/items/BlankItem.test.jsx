import React from 'react';
import renderer from 'react-test-renderer';

import BlankItem from './BlankItem';

describe('BlankItem', () => {
  it('matches its snapshot', () => {
    const component = renderer.create(<BlankItem id="expected-id" />);
    expect(component).toMatchSnapshot();
  });
});
