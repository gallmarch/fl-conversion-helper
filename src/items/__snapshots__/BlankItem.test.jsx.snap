// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlankItem matches its snapshot 1`] = `
<li
  className="empty-icon flch-empty-icon"
  id="expected-id"
/>
`;
